### PR TITLE
Add keyboard shortcut for switching between Messenger and Workchat

### PR DESCRIPTION
**Pull request by eventhandler23**
To easily switch chat environment easily

### CODE REVIEW

**Comment by sindresorhus** [comment]
Those keyboard shortcuts are already taken. You need to use something else. The shortcuts should also be documented in the readme, and you need to add them to both the macOS and Windows/Linux menus.

File: readme.md [hunk 1]
```diff
@@ -156,6 +156,7 @@ Delete conversation    | <kbd>Cmd/Ctrl</kbd> <kbd>Shift</kbd> <kbd>d</kbd>
 Toggle "Always on Top" | <kbd>Cmd/Ctrl</kbd> <kbd>Shift</kbd> <kbd>t</kbd>
 Toggle window menu     | <kbd>Alt</kbd> *(Windows only)*
 Toggle sidebar         | <kbd>Cmd/Ctrl</kbd> <kbd>Shift</kbd> <kbd>s</kbd>
+Switch between Messenger and Workchat  | <kbd>Cmd/Ctrl Shift 1 or 2</kbd>
```
Comment (sindresorhus) [inline readme.md]: Each key needs a separate `kbd` tag.

**Comment by sindresorhus** [comment]
In https://github.com/sindresorhus/caprine/pull/561#discussion_r212820263, I think you misunderstood me. Please look at all the other keyboard shortcuts. Each **keyboard key** has a separate `kbd` tag.

**Comment by Ginkooo** [comment]
@sindresorhus 


image


but there is no Ctrl/Cmd Shift 1/2 combination in here

**Comment by eventhandler23** [comment]
Oh. Now I get it. Sorry about that. Should I make the shortcut for switching one line or still separated?

**Comment by sindresorhus** [comment]
Still separated

**Pull request merged**